### DISCUSSION TITLE
How can I remove the underline text style ?

### DISCUSSION THREAD

**Discussion by joffreypersia** [codingyu/laravel-goto-view#72] [Q&A]
Hi, 

I would like to remove the underline under the blade name. Is it possible ? How ? 

Thank you !

**Comment by codingyu** [comment]
![image](https://github.com/codingyu/laravel-goto-view/assets/20354841/e54aabe6-1bfc-4377-a2ef-3b3cb90f3eda)

**But this should have become non clickable.**

**Reply by joffreypersia** [reply]
Thank you, @codingyu, for your quick answer. 

I want to keep the link clickable but just remove the underline styling. Is it possible?

**Reply by codingyu** [reply]
I haven't come up with a solution yet, it's based on the document link implementation.

**Reply by joffreypersia** [reply]
Thank you :)
Hovewer the fix you privided doesn't fix the issue here has the cmd+click doesn't work anymore
But thank you
